templates/demo-store/remix.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('@remix-run/dev').AppConfig} */
+module.exports = {
+  appDirectory: 'app',
+  ignoredRouteFiles: ['**/.*'],
+  publicPath: (process.env.HYDROGEN_ASSET_BASE_URL ?? '/') + 'build/',
```

Comment:
Is this going to confuse people?

Reply:
Perhaps we can rename it to something else in our CLI? `OXYGEN_CDN`, `SHOPIFY_CDN`?

Reply:
@jplhomer I mean we can add an alias in our CLI until all the other parts are aligned. But I guess we can just stick to `HYDROGEN_ASSET_BASE_URL` for the time being.